estuary/utils/general.py

```diff
@@ -277,68 +277,103 @@ def story_flow(label):
     if label == BugzillaBug.__label__:
         return {
             'uid_name': BugzillaBug.id_.db_property or BugzillaBug.id.name,
+            'display_label': 'Bugzilla bug',
             'forward_relationship': '{0}<'.format(
                 BugzillaBug.resolved_by_commits.definition['relation_type']),
+            'forward_relationship_display': 'resolved by',
             'forward_label': DistGitCommit.__label__,
+            'forward_label_display': 'DistGit commit',
             'backward_relationship': None,
-            'backward_label': None
+            'backward_relationship_display': None,
+            'backward_label': None,
+            'backward_label_display': None
         }
     elif label == DistGitCommit.__label__:
         return {
             'uid_name': DistGitCommit.hash_.db_property or DistGitCommit.hash.name,
+            'display_label': 'DistGit commit',
             'forward_relationship': '{0}<'.format(
                 DistGitCommit.koji_builds.definition['relation_type']),
+            'forward_relationship_display': 'built from',
             'forward_label': KojiBuild.__label__,
+            'forward_label_display': 'Koji build',
             'backward_relationship': '{0}>'.format(
                 DistGitCommit.resolved_bugs.definition['relation_type']),
-            'backward_label': BugzillaBug.__label__
+            'backward_relationship_display': 'resolved by',
+            'backward_label': BugzillaBug.__label__,
+            'backward_label_display': 'Bugzilla bug'
         }
     elif label == KojiBuild.__label__:
         return {
             'uid_name': KojiBuild.id_.db_property or KojiBuild.id.name,
+            'display_label': 'Koji build',
             'forward_relationship': '{0}<'.format(KojiBuild.advisories.definition['relation_type']),
+            'forward_relationship_display': 'attached to',
             'forward_label': Advisory.__label__,
+            'forward_label_display': 'Advisory',
             'backward_relationship': '{0}>'.format(KojiBuild.commit.definition['relation_type']),
-            'backward_label': DistGitCommit.__label__
+            'backward_relationship_display': 'built from',
+            'backward_label': DistGitCommit.__label__,
+            'backward_label_display': 'DistGit commit'
         }
     elif label == Advisory.__label__:
         return {
             'uid_name': Advisory.id_.db_property or Advisory.id.name,
+            'display_label': 'Advisory',
             'forward_relationship': '{0}<'.format(
                 Advisory.triggered_freshmaker_event.definition['relation_type']),
+            'forward_relationship_display': 'triggered by',
```

Comment:
The wording is wrong because this is from the perspective of the Advisory, not the FreshmakerEvent